Saturation/AWPassiveClauseContainer.cpp

```diff
@@ -67,6 +68,9 @@ AWPassiveClauseContainer::AWPassiveClauseContainer(bool isOutermost, const Shell
     _ageRatio = 1;
     _weightRatio = 1;
   }
+
+  _randomize = _opt.randomAWR();
```

Comment:
I would remove the boolean field, `_opt.randomAWR()` will compile to a single load quite happily.